test/player_src_equals_integration.js

```diff
@@ -351,4 +357,22 @@ describe('Player Src Equals', () => {
     // point ensures it is safe to interact with the media element.
     await ready;
   }
+
+  /**
+   * @param {string} contentUri
+   * @param {number} startTime
+   * @return {!Promise}
+   */
+  async function loadWithSrcEqualsAndStartTime(contentUri, startTime) {
+    const ready = new Promise((resolve) => {
+      eventManager.listenOnce(video, 'loadedmetadata', resolve);
```

Comment:
Shouldn't this be `loadeddata`?  That could also cause a race where this resolves before Playhead sets the current time.

Reply:
Yes, it was a mistake. I changed it!